presto-docs/src/main/sphinx/optimizer/statistics.rst

```diff
@@ -41,12 +41,13 @@ The following statistics are available in Presto:
    * **distinct value count**: the number of distinct values
    * **low value**: the smallest value in the column
    * **high value**: the largest value in the column
+   * **histogram**: A connector-dependent histogram data structure.
 
 The set of statistics available for a particular query depends on the connector
 being used and can also vary by table or even by table layout. For example, the
 Hive connector does not currently provide statistics on data size.
 
-Table statistics can be displayed via the Presto SQL interface using the
+Table statistics can be displayed via a SQL statement using the
```

Comment:
```suggestion
Table statistics can be displayed using a SQL statement using the
```
Avoid Latin terms and abbreviations. See the [Gitlab documentation style guide recommended word list entry for via](https://docs.gitlab.com/ee/development/documentation/styleguide/word_list.html#via) for context and options.

Reply:
fixed